buildsystem/HandlePythonOptions.cmake

```diff
@@ -1,16 +1,27 @@
-# Copyright 2015-2018 the openage authors. See copying.md for legal info.
+# Copyright 2015-2020 the openage authors. See copying.md for legal info.
 
 # finds the python interpreter, install destination and extension flags.
 
 # the Python version number requirement is in modules/FindPython_test.cpp
 find_package(Python ${PYTHON_MIN_VERSION} REQUIRED)
+
+# NumPy check
+message(CHECK_START "Looking for NumPy")
+if(NUMPY_FOUND)
+	message(CHECK_PASS "NumPy - found!")
+else()
+	message(FATAL_ERROR "NumPy - not found!")
+endif()
```

Comment:
these messages are at the wrong place i think, they should be within our findpython module. but even there, i think that is already printed during the `findpackage` above? if not, i'd remove the `looking for numpy` message, since we're not really looking for numpy there. if necesary (i.e. if the cmake-findpython-module doesn't abort on its own) we can keep the fatal error message printing in our `findpython.cmake`.